packages/terra-clinical-embedded-content-consumer/src/EmbeddedContentConsumer.scss

```diff
@@ -0,0 +1,15 @@
+.terraClinical-EmbeddedContentConsumer {
+  height: 100%;
+  width: 100%;
+
+  .xfc { // stylelint-disable-line selector-class-pattern
```

Comment:
When you move to css modules, you're going to have to mark this '.xfc' selector as global to prevent css modules from converting it.

{code}
:local {
  .container {
    height: 100%;

    :global {
      .xfc {
        height: 100%;
     
        iframe {
          border: 0;
          height: 100%;
          width: 100%;
        }
      }
    }
  }
}
{code}

Something like that should work.